pkg/abstractions/image/base_requirements.txt

```diff
@@ -14,3 +14,4 @@ psutil==5.9.6
 cloudpickle==3.0.0
 rich==13.7.0
 watchdog==4.0.0
+nest-asyncio==1.6.0
```

Comment:
Can we remove this dep and embed the code directly into sdk/ somewhere?

Reply:
Yep. Added.